pubsub/tests/unit/pubsub_v1/subscriber/test_subscriber_client.py

```diff
@@ -54,3 +54,13 @@ def test_subscribe_with_callback():
         subscription = client.subscribe('sub_name_b', callback)
         open_.assert_called_once_with(callback)
     assert isinstance(subscription, thread.Policy)
+
+
+def test_subscribe_with_failed_callback():
+    import pytest
```

Comment:
@chemelnucfin Please import this at module level (but thanks for trying to adhere to some semblance of a rule)